Driverless AI/ES/introducción-a-machine-learning-automático-con-driverless-ai/introducción-a-machine-learning-automático-con-driverless-ai.md

```diff
@@ -640,75 +640,75 @@ El resumen incluye:
 - **Validation Score** (Puntuación de validación): valor de Pérdida Logarítmica +/- épsilon de la base de la máquina
```

Comment:
I think you can use either "Puntaje" or "Puntuación" but just use one, not both, especially when you're trying to translate the same word "score"

Reply:
Puntaje is used below*